packages/office-ui-fabric-react/src/components/Panel/Panel.styles.ts

```diff
@@ -346,8 +346,8 @@ export const getStyles = (props: IPanelStyleProps): IPanelStyles => {
         transition: `opacity ${AnimationVariables.durationValue3} ${AnimationVariables.easeFunction2}`
       },
       isFooterSticky && {
-        background: palette.white,
-        borderTopColor: palette.neutralLight
+        background: semanticColors.bodyBackground,
+        borderTopColor: semanticColors.variantBorder
```

Comment:
@marygans Do you think `variantBorder` is correct here?

Reply:
Yes this looks good! (Also checked with design for confirmation)